### PR TITLE
New package: lms-3.40.0

### PR DESCRIPTION
**Pull request by Danoloan10**
LMS is a lightweight Subsonic compatible music server written in C++. It is developed using the Wt web framework.

#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [x] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [x] i686
  - [x] x86_64-musl
  - [x] aarch64
  - [x] aarch64-musl
  - [x] armv7l
  - [x] armv7l-musl
  - [x] armv6l
  - [x] armv6l-musl

### CODE REVIEW

File: srcpkgs/lms/files/lms/run
Comment (Duncaen) [inline srcpkgs/lms/files/lms/run:4]: Not exactly sure, but `USER` is usually already set, should probably use some different variable name or reset it before sourcing the conf file. Lets use `LMSUSER` to match other packages.

File: srcpkgs/lms/files/lms/run
Comment (Duncaen) [inline srcpkgs/lms/files/lms/run:5]: This is the default so I would just keep it out.

File: srcpkgs/lms/files/lms/run
Comment (Duncaen) [inline srcpkgs/lms/files/lms/run:6]: This path looks wrong, should probably be `/var/lms`.

File: srcpkgs/lms/files/lms/run
Comment (Duncaen) [inline srcpkgs/lms/files/lms/run:10]: I would suggest to use this:
```shell
#!/bin/sh

[ -r conf ] && . ./conf
exec 2>&1
exec chpst -u "${LMSUSER:-_lms}" -C "${WORKDIR:-/var/lms}" lms ${OPTS}
```

**Review by Duncaen — CHANGES_REQUESTED** [review]

**Comment by Danoloan10** [comment]
Thanks for the review! Will fix asap.

**Comment by Duncaen** [comment]
Awesome, I'll merge it when you are ready. This has been sitting around for too long and you have been updating this consistently.

File: srcpkgs/lms/files/lms/run
Comment (Danoloan10) [reply srcpkgs/lms/files/lms/run]: Just did what you said here, pretty much agree :)

File: common/shlibs
Comment (Duncaen) [inline common/shlibs:103]: Addubg the glib shared libaries here again looks wrong.

Comment (Danoloan10) [reply common/shlibs]: It is :( Git tricked me rebasing

**Comment by Danoloan10** [comment]
It seems like LMS added a new dependency, I will test the builds locally again before uploading a new patch just to make sure there are no other new errors.

**Comment by Danoloan10** [comment]
I have tested it both in x86_64-gnu and aarch64-musl, I think it should work now

**Review by Duncaen — APPROVED** [review]

**Comment by Danoloan10** [comment]
I think the failures may be due to dirty workspaces or something like that, I got them but they dissappeared after purging the binpkgs cache.

**Comment by Duncaen** [comment]
> I think the failures may be due to dirty workspaces or something like that, I got them but they dissappeared after purging the binpkgs cache.

Yes I was just waiting on those builds that are "breaking" the CI to finish.

**Pull request merged**
